mifosng-android/build.gradle

```diff
@@ -4,6 +4,7 @@
  */
 
 buildscript {
+    ext.kotlin_version = "1.3.31"
```

Comment:
Please make a separate PR for Kotlin support.

Reply:
Ok sure.